docs/products/kafka/kafka-connect/concepts/jdbc-source-modes.rst

```diff
@@ -0,0 +1,125 @@
+JDBC source connector modes
+===========================
+
+
+JDBC source connector extracts data from a relational database, such as PostgreSQL or MySQL, and pushes it to Apache Kafka where can be transformed and read by multiple consumers. The details of the connector are covered in the `Aiven JDBC source connector GitHub documentation <https://github.com/aiven/jdbc-connector-for-apache-kafka/blob/master/docs/source-connector.md>`_. 
+
+This connector type periodically queries the table(s) to extract the data, and can be configured in four **modes**.
+
+Bulk mode
+---------
+
+In ``bulk`` mode the connector will periodically query the full table retrieving all the rows and publishing them into the Apache Kafka topic.
+Thus, if the source table contains ``100.000`` rows, the connector will insert ``100.000`` new messages in the Apache Kafka topic for every poll, no matter how many rows in the database are new or stale.
+
+.. Tip::
+
+  Since the bulk mode replicates the whole table content into the Apache Kafka topic at every poll, it's a suitable option only for tables with limited amount of data which don't have any incremental or timestamp column.
+
+Incrementing mode
+-----------------
+
+Using the ``incrementing`` mode, the connector will query the table appending a `WHERE` condition based on a **incrementing column** to fetch the new rows. The incrementing mode requires the presence in the source table of a column containing an always growing number (like a series). The incrementing column is used to check which rows have been added since last query. 
```

Comment:
```suggestion
Using the ``incrementing`` mode, the connector will query the table and append a `WHERE` condition based on an **incrementing column** in order to fetch new rows. The incrementing mode requires that a column containing an always growing number (like a series) is present in the source table. The incrementing column is used to check which rows have been added since last query. 
```